R/write.R

```diff
@@ -67,7 +68,8 @@ xportr_write <- function(.df,
   domain <- domain %||% attr(.df, "_xportr.df_arg_")
   if (!is.null(domain)) attr(.df, "_xportr.df_arg_") <- domain
 
-  metadata <- metadata %||% attr(.df, "_xportr.df_metadata_")
+  # metadata should not be inferred from the data frame if it is not provided
+  # by the user.
```

Comment:
Alternatively, we could infer \*only\* if it's a `metacore` instance.

```suggestion
  # metadata should only be inferred from the data frame if it is a `Metacore` object
  metadata_tmp <- metadata %||% attr(.df, "_xportr.df_metadata_")
  if (inherits(metadata_tmp, "Metacore")) metadata <- metadata_tmp
```